src/mqtt_rpc_client_qt.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include <QJsonDocument>
 #include <QSslConfiguration>
-#include <QTextCodec>
+#include <QtCore5Compat/QTextCodec>
```

Comment:
As far as I can tell, QTextCodec is only used in one single line:
`emit command_result(*command, QTextCodec::codecForMib(106)->toUnicode(payload));`

But, codecForMib(106) is UTF-8 according to https://codebrowser.dev/qt5/qtbase/src/corelib/codecs/qtextcodec.cpp.html#1079 and https://www.iana.org/assignments/character-sets/character-sets.xhtml

So, I think you can remove the qt5compat linkage, remove inclusion of QTextCodec, and just use `QString::fromUtf8(payload)` - although I may be wrong...

Reply:
Thanks, I'll use master, and close this pull request